go/geth_adapter/adapter_test.go

```diff
@@ -27,6 +32,12 @@ type StateDb interface {
 	geth.StateDB
 }
 
+func TestRunContextAdapter_GethInterpretersIsAvailable(t *testing.T) {
```

Comment:
nit: this is not so much a property of the `geth_adapter` package but the `geth` package imported above using
```
_ "github.com/Fantom-foundation/Tosca/go/interpreter/geth"
```
So is likely not testing anything of the package, just the effect of the import above.

Reply:
Right, I removed the test.